docs/language/informal/toplevel-inference.md

```diff
@@ -0,0 +1,338 @@
+﻿# Toplevel inference
+
+Owner: leafp@google.com
+
+Status: Proposal
+
+
+## Top level inference overview
+
+Top level inference has two phases:
+
+1. **Method override inference**
+    * If you omit a return type or parameter type from an overridden method,
```

Comment:
OK
The exact meaning of `does not agree` was not clear.
Now I understand what it means.
Thank you.